examples/3Scaling/2ParallelDatabase/pDB14_AdjBFSTEST.m

```diff
@@ -23,13 +23,13 @@
 
 if DoDB
     DBsetup;                          % Create binding to database.
-    G = DBaddJavaOps('edu.mit.ll.graphulo.MatlabGraphulo','instance','localhost:2181','root','secret');
+    %G = DBaddJavaOps('edu.mit.ll.graphulo.MatlabGraphulo','instance','localhost:2181','root','secret');
     
     rnameMat = [getName(Tadj) '_Mat_k' num2str(kmax)];
     TresMat = DB(rnameMat);
     deleteForce(TresMat);
     TresMat = DB(rnameMat);
-    G.addAllSumCombiner(rnameMat);
+    DB.addAllSumCombiner(rnameMat);
```

Comment:
I guess it's old code that got removed at some point.  If its purpose is to add a summing combiner to `TresMat`, then the correct method is [`addColCombiner`](https://github.com/Accla/d4m/blob/master/matlab_src/%40DBtable/addColCombiner.m). Could you verify?